docs/api-testing/composer.md

```diff
@@ -369,6 +369,150 @@ To convert a timestamp from a payload response to milliseconds:
+
+These methods can be used anywhere you can write a variable, such as inside a SET (variable) or in any part of the request (body, header, param, etc.).
+
+### Set (variable)
+Set (variable) allows you to create variables or more structured data.
+
+To create a single variable:
+
+* **Var:** The name to assign the variable
+* **Variable Mode:** String
+* **Value:** The method to use to generate data. For example, `${F.fullName()}` will generate a random full name.
+
+<img src={useBaseUrl('/img/api-testing/test-data-full-name.png')} alt="Full name generation" width="600"/>
+
+To create an array of data:
+
+* **Var:** The name to assign the variable
+* **Variable Mode:** Data
+* **Data:** The JS function for creating an array. For example, `new Array(5).fill(0).map(_ => F.streetAddress())` generates an array with five random addresses.
+
+<img src={useBaseUrl('/img/api-testing/test-data-array.png')} alt="Data array generation" width="600"/>
+
+To create an object of data:
+
+* **Var:** The name to assign the variable
+* **Variable Mode:** Language
+* **Lang:** Template
+* **Content:** The object to generate with the required methods.
+
+```
+{
+"name": "${F.firstName()}",
+"last name": "${F.lastName()}",
+"address": "${F.streetName()}",
+"profession": "${F.profession()}",
+"mobile phone": "${F.mobile()}",
+"email": "${F.emailAddress()}"
+}
+```
+<img src={useBaseUrl('/img/api-testing/test-data-personal-data.png')} alt="Personal data generation" width="600"/>
+
+### Body
+
+Any of the following methods can be used in a request body.
+
+* **Content-Type:** The content-type of the body (application/json in this example)
+* **Content:** The body of the request.
+
+```
+{
+"name": "${F.firstName()}",
+"last name": "${F.lastName()}",
+"city": "${F.city()}",
+"profession": "${F.profession()}"
+}
+```
+
+<img src={useBaseUrl('/img/api-testing/test-data-request-body.png')} alt="Request body data" width="600"/>
+
+In this example we use a POST body, but this can be applied in all REST methods. Similarly, these methods can also be used as params.
+
+* **Name:** The name of the param
+* **Value:** The value of the param. For example, `${F.creditCardNumber()}`
+
+<img src={useBaseUrl('/img/api-testing/test-data-request-body-param.png')} alt="Request body param data" width="600"/>
+
+These examples are of the most common places where you may need to generate data, but these methods can be added anywhere you can use a variable.
+
+### Test Data Methods
+
+#### Addresses and Countries
+* `F.streetName()` - Generates a street name
+* `F.streetAddressNumber()` - Generates an address number
+* `F.streetAddress()` - Generates a street and address number. If secondary is specified, this method provides an apartment number.
+* `F.secondaryAddress()` - Generates an apartment number
+* `F.zipCode()` - Generates a ZIP code. If a state has been provided, a proper ZIP code for the state is provided. Valid only for US states.
```

Comment:
@lauramthomas remove this statement :"If a state has been provided, a proper ZIP code for the state is provided"